### PR TITLE
Adding Ludo-The_Game project

### PR DESCRIPTION
**Pull request by adi-g15**
I have added my project [Ludo-The_Game](https://github.com/AdityaGupta150/Ludo-The_Game). 
I have added some issues that are beginner-friendly for the starters to work on, specially for coming hacktoberfest. They are simple and I will guide with futher clarifications for pull requests too.

### CODE REVIEW

File: _data/projects/ludo-the-game.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  - board-game
+  - algorithm
+  - html
+  - multi-threaded
+  - terminal
+  - css
+  - reactjs
+  - angular
+upforgrabs:
+  name: hacktoberfest
```
Comment (ritwik12) [inline _data/projects/ludo-the-game.yml]: @AdityaGupta150 Typo here, your label is `Hacktoberfest` as per link.

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/ludo-the-game.yml (hunk 1 above)
Comment (adi-g15) [reply _data/projects/ludo-the-game.yml]: @ritwik12 Corrected the typo.

**Review by adi-g15 — COMMENTED** [review]
Fixed typo

File: _data/projects/ludo-the-game.yml (hunk 1 above)
Comment (ritwik12) [reply _data/projects/ludo-the-game.yml]: Thanks

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@AdityaGupta150 Thanks!!

**Pull request merged**